python/pydantic_core/core_schema.py

```diff
@@ -91,6 +92,8 @@ class CoreConfig(TypedDict, total=False):
     ser_json_bytes: Literal['utf8', 'base64']  # default: 'utf8'
     # used to hide input data from ValidationError repr
     hide_input_in_errors: bool
+    # used for plugins to add custom settings
+    plugin_settings: dict[str, Any]
```

Comment:
I think failures are related to the syntax not working on older Python versions:

```suggestion
    plugin_settings: Dict[str, Any]
```